test/verify/check-machines

```diff
@@ -212,6 +212,8 @@ class TestMachines(MachineCase):
         b.wait_present("#test-vm-action-{0}".format(self.providerVm1_name)) # is provider-specific action button rendered?
         b.wait_in_text("#test-vm-props-{0}".format(self.providerVm1_name), "Test Provider Property") # are provider-specific VM props rendered?
 
+        b.click("{0}-sendNMI".format(self.providerVm1_id)) # click the Send Non-Maskable Interrupt button
```

Comment:
@mareklibra @petervo @stefwalter, I moved the sendNMI test from testProvider to testBasic. 
Regarding expanding the test, since the tested VM is configured to run without an OS, then sending NMI does not take any effect on the tested VM. The kernel should handle this interrupt and if there is no kernel then the interrupt is ignored. 
So I think that either we leave the test as is for covering at least cockpit-machines internal processing or we remove the test at all.